test/bigtest/tests/jobs-test.js

```diff
@@ -41,30 +43,30 @@ describe('Jobs lists', () => {
     });
 
     it('should display correct amount of running jobs', () => {
-      expect(runningJobs.jobItemsAmount).to.equal(3);
+      expect(runningJobs.jobItemsAmount).to.equal(2);
     });
 
     it('should display running job', () => {
       expect(runningJobs.jobItems(0).isPresent).to.be.true;
     });
 
     it('should display correct job profile name', () => {
-      expect(runningJobs.jobItems(0).jobProfileName).to.equal(sampleJobExecution.jobProfileInfo.name);
+      expect(runningJobs.jobItems(0).jobProfileName).to.equal(runningJobExecution.jobProfileInfo.name);
     });
 
     it('should display correct file name', () => {
-      expect(runningJobs.jobItems(0).fileName).to.equal(sampleJobExecution.fileName);
+      expect(runningJobs.jobItems(0).fileName).to.equal(runningJobExecution?.exportedFiles?.[0]?.fileName);
```

Comment:
As far as I understand, runningJobExecution, exported files and the first file are always presented. Could you please clarify why do we need to use ?. syntax? Or do i misunderstand the idea?
In line 54 this syntax is not used.

Reply:
Will try that. I had some issues with that, so added it.